internal/test/e2e/tinkHwQueue.go

```diff
@@ -0,0 +1,29 @@
+package e2e
+
+import (
+	"sync"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+)
+
+type hwQueue struct {
+	hws []*api.Hardware
+	mu  sync.Mutex
```

Comment:
What's the mutex being used for? Is this queue supposed to be thread safe?

Reply:
Yes, its a concurrent queue since multiple Goroutines tries to read and write from the queue it needs to be thread safe!